sql/core/src/main/scala/org/apache/spark/sql/execution/WholeStageCodegenExec.scala

```diff
@@ -522,13 +517,7 @@ case class InputAdapter(child: SparkPlan, isChildColumnar: Boolean)
     child.executeColumnar()
```

Comment:
`InputAdapter` doesn't support columnar execution now? Seems we can change `supportsColumnar` and remove `doExecuteColumnar`?